roles/matterbridge/vars/vault.yml

```diff
@@ -1,10 +1,76 @@
+65666332636633643535643365626637336566336438396364396436376631613132653239313939
+64616662356136396466623861353563313062333665313135386338383033306532653663643933
+35356566386434313666386263656530616137366564643566383137346632636464346463383238
+31313766663461613561393933633363386637613936323535633831333530393534316163313535
+33363238333438326430356137336161363563623538333735363330356339623364666536346537
+32313466326463306262333539346238386366393163303335376366333931656166653164623632
+33393230633138643935623465353132393636393735336337316236356230616439363439353839
+35333734646231383232393735663531396262376533316665323336313239396331366136346630
+66633666353966373731336464636636663162386139366161306439353839303834646539386161
+65643831393761653033393864396363366663323266393531633065396365363339393336316265
+61353734633735616263393634623965303239653938333934633965616631653862343465653934
+39353862656565303566386236396330373463353565343966303061663036646565653532663135
+65633632346661313932383230646437343339663630646565373832636539303762636130306461
+32613664623436626435373965383731323237313962393165313636626339656134356432646234
+34393130663236313735333034366261313466376334643866613037616438653863643331623262
+31316633393033393633653464316138376337303430353933623866653162613337636533643132
+30366537626662353831666663383232353363386231383238323861373366363734323835396233
+35386532353166346164623762366261393733306630386561653436356238633664346332663232
+38343537373766336465316163393961353234303039333431393532663230376638633033373463
+64646265313061633336343364613838336335366531343639666562353766626433306431306361
+63643236636562626537396161363738393333323938643739363235323362393435656666313266
+33383132393937356337356537333461663735343638653562663565623863353433623431316434
+31346533336463613164356531663638343237633865636130376232373563663439366537646236
+37346535326332353032656561346335636234396435386461313031346136333936303963363732
+64383231363931323233663866343965643237383936376366323733363233383566643639376238
+66336638376262306636393764336638623833613364343934386230386635363162353763653734
+64313034343862373964336466626464356163366364366637326262396639343836613633666333
+32333566613237623136303363363835363862356562336232663961363031313537316365626336
+62396331643032373135616639326466323064613638343864336436356232303262386337623532
+36636530633364383166336635373239353531343934333162636539336631313431653465343565
+37383232303432386439323337646362333239303635323562643066613238316462313765336665
+66643561373838613837303866386261656262613732666639306132333637336163646465646538
+36626465613033386130383039663764623439393935666339653034663562333334613132336364
+61306437613531643963616331346635303065656332303334306436636365613664333637633336
+64356138653436386537366361663165643666323334383461636338623232393864626463333131
+66306166623133636634626336326563333562613464396363373963663830653763323932343134
+613839383830633331343962376362643239
```

Comment:
This looks like a lot of new data. I should probably take a look at this locally. 😀

Reply:
its just all the tokens and stuff from the FOSSRIT bridges (the ones i already sent you via proton pass) - so Telegram, IRC, slack, matrix, etc as well as some various other config

Reply:
LGTM. 👍🏻